nixos/modules/system/boot/systemd.nix

```diff
@@ -749,6 +749,22 @@ in
       '';
     };
 
+    systemd.tmpfiles.packages = mkOption {
+      type = types.listOf types.package;
+      default = [];
+      example = literalExample "[ pkgs.lvm2 ]";
+      apply = map getLib;
+      description = ''
+        List of packages containing <command>systemd-tmpfiles</command> rules.
+
+        All found files in
+        <filename><replaceable>pkg</replaceable>/lib/tmpfiles.d</filename>
+        will be included.
```

Comment:
I'd add "If none of these files can be found, this will return an error."

Reply:
Hm, right now we actually check if this folder exist, not if any files are in there. Maybe we should change that check (in addition to documenting it).